### PR TITLE
docs: tells users where they are

### PR DESCRIPTION
**Pull request by iamayushdas**
Please answer these questions before submitting a pull request

- Why submit this pull request?
* [ ] Bugfix
* [x] New feature provided
* [ ] Improve performance
* [ ] Backport patches
___
### Bugfix
- Description
tells users where they are in sidebar
- How to fix?
added a label of Apache Apisix Dashboard to sidebar 
___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch
https://github.com/iamayushdas/apisix-dashboard/tree/sidebar

- Related commits and pull requests
https://github.com/apache/apisix-website/issues/230

- Target branch
https://github.com/apache/apisix-dashboard/tree/master

### CODE REVIEW

**Comment by iamayushdas** [comment]
@juzhiyuan @guoqqqi @LiteSun @gxthrj have a look

**Review by gxthrj — COMMENTED** [review]
Sorry, but I am not very good at front-end reviews.

**Comment by guoqqqi** [comment]
Hi, @iamayushdas If we are sure to use Configure to replace Edit, the Chinese version of the Upstream page has not yet been replaced with Configure.
Then I found a small problem when editing the page, the title should be `Edit **`, but now when editing it is `Create **`.
image
Thank you.

**Comment by iamayushdas** [comment]
> Hi, @iamayushdas If we are sure to use Configure to replace Edit, the Chinese version of the Upstream page has not yet been replaced with Configure.
> 
> Then I found a small problem when editing the page, the title should be `Edit **`, but now when editing it is `Create **`.
> 
> image
> 
> Thank you.
> 
> 

Okay let me look into this

**Comment by iamayushdas** [comment]
> > Hi, @iamayushdas If we are sure to use Configure to replace Edit, the Chinese version of the Upstream page has not yet been replaced with Configure.
> > Then I found a small problem when editing the page, the title should be `Edit **`, but now when editing it is `Create **`.
> > image
> > Thank you.
> 
> Okay let me look into this

@juzhiyuan after looking into the fix raised by @guoqqqi this, i got to know many of the routings was wrongly done like "Service and upstream". parameters were not set correctly. so should i fix this in the same PR or should i raise an issue?

**Comment by iamayushdas** [comment]
@juzhiyuan should i change the path name of edit to configure for all or not, i guess that would be good. What say?

**Comment by juzhiyuan** [comment]
I would prefer only to update the Title, keep the path as `/create/edit/`, it's more clear for users to know what happened to this URL.

As for the title, `Create Route` -> `/create`, `Configure Route` or `Edit Route` or `Update Route` -> `/edit` are ok for me.

**Comment by iamayushdas** [comment]
Okay

**Comment by iamayushdas** [comment]
@juzhiyuan have a check

File: docs/en/latest/config.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
   "version": 2.4,
   "sidebar": [
+    {
+      "type": "link",
+      "label": "Apache Apisix Dashboard",
```
Comment (juzhiyuan) [inline docs/en/latest/config.json:6]:
```suggestion
      "label": "Apache APISIX Dashboard",
```

File: web/src/locales/zh-CN/component.ts [hunk 2]
```diff
@@ -21,19 +21,24 @@ export default {
   'component.global.search': '查询',
   'component.global.reset': '重置',
   'component.global.confirm': '确认',
+  'component.global.format': '格式',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/component.ts:24]:
```suggestion
  'component.global.format': '格式化',
```

File: web/src/locales/zh-CN/component.ts [hunk 3]
```diff
@@ -21,19 +21,24 @@ export default {
...
+  'component.global.document': '文档',
+  'component.global.enable': '使能够',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/component.ts:26]:
```suggestion
  'component.global.enable': '启用',
```

File: web/src/locales/zh-CN/component.ts [hunk 4]
```diff
@@ -21,19 +21,24 @@ export default {
...
+  'component.global.scope': '范围',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/component.ts:27]:
```suggestion
  'component.global.scope': '作用域',
```

File: web/src/locales/zh-CN/component.ts [hunk 5]
```diff
@@ -21,19 +21,24 @@ export default {
...
+  'component.global.data.editor': '资料编辑器',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/component.ts:28]:
```suggestion
  'component.global.data.editor': '元数据编辑器',
```

File: web/src/locales/zh-CN/component.ts [hunk 6]
```diff
@@ -21,19 +21,24 @@ export default {
...
   'component.global.delete': '删除',
   'component.global.cancel': '取消',
   'component.global.submit': '提交',
   'component.global.create': '创建',
   'component.global.createWithEditor': '使用编辑器创建',
   'component.global.add': '新建',
   'component.global.save': '保存',
-  'component.global.edit': '编辑',
+  'component.global.edit': '配置',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/component.ts:36]: actually, it's more clear to use `编辑` 😂

```suggestion
  'component.global.edit': '编辑',
```

File: web/src/pages/Plugin/PluginMarket.tsx [hunk 7]
```diff
@@ -20,6 +20,7 @@ import { PageHeaderWrapper } from '@ant-design/pro-layout';
 
 import PluginPage from '@/components/Plugin';
 import { fetchList, createOrUpdate } from './service';
+import { formatMessage } from 'umi';
```
Comment (juzhiyuan) [inline web/src/pages/Plugin/PluginMarket.tsx:23]: ok, we need to use

```suggestion
import { useIntl } from 'umi';
```

here, and use `const { formatMessage } = useIntl()`

File: web/src/pages/Plugin/locales/en-US.ts [hunk 8]
```diff
@@ -16,5 +16,7 @@
  */
 export default {
   'page.plugin.drawer.popconfirm.title.delete': 'Are you sure to delete this item?',
-  'page.plugin.list': 'Plugin List'
+  'page.plugin.list': 'Plugin List',
+  'page.plugin.list.enabled': 'List of enabled plugins',
+  'page.plugin.market.config': 'Config Plugin',
```
Comment (juzhiyuan) [inline web/src/pages/Plugin/locales/en-US.ts:21]: Not sure to use `config` or `configure`?

File: web/src/pages/Service/Create.tsx [hunk 9]
```diff
@@ -116,7 +116,7 @@ const Page: React.FC = (props) => {
     <>
       <PageHeaderWrapper
         title={`${
-          (props as any).match.params.rid
+          (props as any).match.params.serviceId
```
Comment (juzhiyuan) [inline web/src/pages/Service/Create.tsx:119]: Do we have `serviceId`? cc @nic-chen

**Review by juzhiyuan — COMMENTED** [review]
😄

File: web/src/pages/Service/Create.tsx (hunk 9 above)
Comment (nic-chen) [reply web/src/pages/Service/Create.tsx:119]: > Do we have `serviceId`? cc @nic-chen 

looks like use in creating a service? if yes, we do need it here

Comment (iamayushdas) [reply web/src/pages/Service/Create.tsx:119]: yes , in code `serviceId` was defined in place of `rid`. so i have to do this in order to change the create title to configure 
title while clicking on configure. is that okay now? @juzhiyuan

Comment (juzhiyuan) [reply web/src/pages/Service/Create.tsx:119]: ok :)

**Review by juzhiyuan — APPROVED** [review]
LGTM!

**Review by LiteSun — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Comment by iamayushdas** [comment]
@liuxiran have a view to this

**Review by nic-chen — APPROVED** [review]

**Pull request merged**